Dockerfile

```diff
@@ -1,17 +1,21 @@
-FROM lukecampbell/ckan
+FROM ckan/ckan
 
 # Install git
 RUN DEBIAN_FRONTEND=noninteractive apt-get update -y
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -q -y git libgeos-dev libxml2-dev libxslt1-dev supervisor postgresql-client
 
 # Install the CKAN Spatial extension
-RUN $CKAN_HOME/bin/pip install -e git+https://github.com/lukecampbell/ckanext-spatial.git@deploy#egg=ckanext-spatial
+# CKAN spatial extension has no tagged Git releases currently, so freeze the
+# version at a known good commit to prevent breakage from later versions
+RUN $CKAN_HOME/bin/pip install -e git+https://github.com/ckan/ckanext-spatial.git@#egg=ckanext-spatial
```

Comment:
Can you put in a commit sha here to freeze it to a version we know is good? This will install with a certain variable ruleset, I woulud rather freeze to a certain commit.

I think it goes:
- Latest release on master
- Anything marked "public"
- Anything marked "stable"
then finally
- master